### ISSUE TITLE
Weaken/remove warning in modeling docs for v1.0?

### ISSUE THREAD

**Issue by eteq** [astropy/astropy#3336]
I noticed that `modeling`'s docs still have the "astropy.modeling is currently a work-in-progress, and thus it is likely there will still be API changes" warning.  Should this be modified for v1.0 to indicate that we think it's going to be fairly stable going forward, but with enhancements (if that's true...)?

@embray @nden @astrofrog

**Comment by mhvk** [comment]
I'd think that a warning is still reasonable, given that quantities are not yet supported. That said, the API looks fantastic and I'm looking forward to trying it! Maybe rephrase in terms of "hope this is it, feedback now very much appreciated?"

**Comment by embray** [comment]
I feel like I already weakened it a bit in #3231, but it could probably be weakened further.

@mhvk I don't think Quantity support is too relevant in terms of API stability.  It will be an enhancement, yes, but it won't significantly change how models are worked with in general.  More likely bigger changes include changes to fitters (a process I already started a long time ago but was shelved due to higher priority issues), and further updates to details of the compound model framework, which is still somewhat experimental I think--there again the basics aren't likely to change much, but some specifics might.

**Comment by mhvk** [comment]
@embray - agree that no major changes should be necessary, but given the issues we had with `Column` and the resulting introduction of `QTable`, I prefer avoiding premature claims of API stability.

**Comment by embray** [comment]
I don't think it's premature.  The basic interface hasn't changed since 0.3.  Any changes that do occur will be around the margins, and will retain backwards compatibility.

**Comment by astrofrog** [comment]
If @embray feels the API is going to be stable, then I'm all in favor of removing the warning. We can still not claim it's stable in the stability page, but the warning is currently a bit strong.

**Comment by eteq** [comment]
:+1: to @astrofrog's suggestion if we are willing to commit to what @embray says here.  I think the warning is only necessary if backwards-incompatible changes are likely.

Another possibility might be to change it to a ``.. note::`` that says something like "backwards compatibility is likely to be maintained, but significant new features may still be coming in future versions." (which is pretty close to what the warning says right now.)

**Comment by astrofrog** [comment]
Yes, we should at *least* change it to just a note instead of a big red warning

**Comment by embray** [comment]
Anything to not scare anyone away.  One of my biggest frustrations with working on this package has simply that it has been underutilized and not tested enough outside a handful of people (granted the feedback of people who have been using it has been invaluable, but it hasn't gone much outside the Astropy developer community...)

**Comment by mhvk** [comment]
@embray - fair enough, we should encourage people to use it!  Happy with the `.. note::` approach, just want to ensure we're not stuck if a (minor) API change does turn out to be required.

**Comment by astrofrog** [comment]
@eteq - will you make this change?

**Comment by embray** [comment]
Assuming it hasn't already been done, I will do it alongside my "what's new" entry for the modeling package.

**Comment by astrofrog** [comment]
@embray - ok, sounds good, the what's new entry still needs doing.
